atomics/T1055/T1055.yaml

```diff
@@ -3,26 +3,25 @@ attack_technique: T1055
 display_name: Process Injection
 
 atomic_tests:
-- name: Process Injection via mavinject.exe
+- name: Process Injection Via Five Techniques Single EXE
```

Comment:
```suggestion
- name: Process Injection via five techniques using a single binary
```

Reply:
Correct, I thought I had structured the files that way. I'll confirm and fix